### ISSUE TITLE
Unhandled error when opening sitemap.xml

### ISSUE THREAD

**Issue by MrMooky** [pecotamic/sitemap#8]
I get this error when opening the sitemap.xml on my server:

```
Unhandled error
TypeError: Argument 1 passed to Pecotamic\Sitemap\Models\SitemapEntry::__construct() must be of the type string, null given, called in /home/privy/vendor/pecotamic/sitemap/src/Models/Sitemap.php on line 48
Location
vendor/pecotamic/sitemap/src/Models/SitemapEntry.php:13 - Pecotamic\Sitemap\Models\SitemapEntry::__construct
```

Dumped the output from the vendor file, got this:

<img width="408" alt="Screenshot 2021-05-08 at 19 45 03" src="https://user-images.githubusercontent.com/3603346/117548569-f8de3700-b035-11eb-98dd-cc325b813a59.png">

PHP 7.4 and latest Laravel and Statamic.

**Comment by snoopy21** [comment]
I get the same error too.

PHP 8.0.5 and latest Laravel and Statamic.

**Comment by werner-freytag** [comment]
Fixed in hotfix release 1.2.2
